### PR TITLE
Styling changes to the map + added a list section

### PR DESCRIPTION
**Pull request by maisiembleton**

### CODE REVIEW

**Review by malinmalliyawadu — CHANGES_REQUESTED** [review]

File: src/components/List/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  border-bottom: 3px solid #8b3032;
+`;
+
+const List = () => {
+  return (
+    <div>
+      <ListHeader>FIND YOUR CLOSEST STOCKIST</ListHeader>
+      <ListSection category="Countdown" />
+      <ListSection category="New World" />
+      <ListSection category="Moore Wilsons" />\
```
[the remark this answers is not in the document]
Comment (maisiembleton) [reply src/components/List/index.tsx:16]: yeah idk feel like this isn't the best way

[the remark this answers is not in the document]
Comment (maisiembleton) [reply src/components/List/index.tsx:16]:
```suggestion
      <ListSection category="Moore Wilsons" />
```

File: src/components/Sidebar/index.tsx [hunk 2]
```diff
@@ -20,37 +21,37 @@ const FiltersBox = styled.div`
...
 export const Sidebar = () => (
   <SidebarWrapper>
     <Search />
 
     <FiltersBox>
-      {/* <FiltersLabel>Filters:</FiltersLabel>
+      <FiltersLabel>Filters:</FiltersLabel>
       <FilterButton>
-        <Check />
+        {/* <Check /> */}
```
[the remark this answers is not in the document]
Comment (maisiembleton) [reply src/components/Sidebar/index.tsx:44]: what is it for

[the remark this answers is not in the document]
Comment (malinmalliyawadu) [reply src/components/Sidebar/index.tsx:44]: dunno - can delete

**Review by malinmalliyawadu — APPROVED** [review]

**Pull request merged**